Harvest.Web/ClientApp/src/FormValidation/UseInputValidator.tsx

```diff
@@ -29,7 +48,9 @@ export function useInputValidator<T>(
 
   useEffect(() => {
     const errorCount = Object.values(errors).filter(notEmptyOrFalsey).length;
-    const previousErrorCount = Object.values(previousErrors || {} as Record<TKey, string>).filter(notEmptyOrFalsey).length;
+    const previousErrorCount = Object.values(
+      previousErrors || ({} as Record<TKey, string>)
```

Comment:
This added an extra ( but I think that is correct?

Reply:
Yeah, prettier likes to put casts inside parenthesis.